packages/jellyfish-transaction-builder/src/txn/txn_builder_governance.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  OP_CODES, Script, TransactionSegWit,
+  CreateProposal
+} from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+import { TxnBuilderError, TxnBuilderErrorType } from './txn_builder_error'
+import BigNumber from 'bignumber.js'
+
+export class TxnBuilderGovernance extends P2WPKHTxnBuilder {
+  /**
+   * Creates a Community fund proposal.
+   *
+   * @param {CreateProposal} createCfp txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+  async createCfp (createCfp: CreateProposal, changeScript: Script, network = 'regtest'): Promise<TransactionSegWit> {
```

Comment:
can help change default reg to mainnet?? i will correct mine later
```suggestion
  async createCfp (createCfp: CreateProposal, changeScript: Script, network = 'mainnet'): Promise<TransactionSegWit> {
```